fury/optpkg.py

```diff
@@ -44,7 +44,8 @@ class TripWire(object):
     ...     import silly_module_name
     ... except ImportError:
     ...    silly_module_name = TripWire('We do not have silly_module_name')
-    >>> silly_module_name.do_silly_thing('with silly string') #doctest: +IGNORE_EXCEPTION_DETAIL
+    >>> silly_module_name.do_silly_thing('with silly string') #doctest:
+                                +IGNORE_EXCEPTION_DETAIL
```

Comment:
Unfortunately, you can not change this line. it will make the test fail.

Reply:
Great, I'll add more test

Reply:
@skoudoro I've changed the line back to default, but still, the test sare failing, what do you think could be the issue?